kmir/k-src/mir-lexer-syntax.md

```diff
@@ -9,13 +9,17 @@ This module defined the necessary `token` productions.
                       | LocalToken
                       | BBId
                       | DoubleHexDigitNoIntLetter
-                      | Whitelisted
   
-  syntax Whitelisted ::= "main" [token]
+  syntax IdentifierToken ::= "main" [token]
```

Comment:
It's worth also pushing these token productions down into the semantics modules that actually use them (i.e. declare the token for `main` alongside the rule / configuration that needs to mention `main`)